modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -376,18 +419,27 @@ private function add_icons_controls(): void {
 				'condition'   => [
 					'icon_platform' => [
 						'Telephone',
-						'Telegram',
+						'Messenger',
 						'WhatsApp',
 					],
 				],
 			],
 		);
+
+		$platform_icons_js = json_encode( self::$platform_icon_mapping );
+
 		$this->add_control(
 			'icon',
 			[
 				'type'          => Controls_Manager::REPEATER,
 				'fields'        => $repeater->get_controls(),
-				'title_field'   => "<i class='{{{ icon_icon.value }}}' ></i> {{{ icon_platform }}}",
+				'title_field'   => "
+<#
```

Comment:
maybe extract this to a function?
private get_icon_title_field()

Reply:
@TzviRabinovitch Done.